dokka-runners/dokka-gradle-plugin/src/main/kotlin/formats/DokkaPublication.kt

```diff
@@ -38,32 +46,116 @@ constructor(
+    /**
+     * The output directory for the finished Dokka publication.
+     *
+     * Default: a subdirectory named [formatName] inside of
+     * [org.jetbrains.dokka.gradle.DokkaExtension.dokkaPublicationDirectory].
+     */
     abstract val outputDirectory: DirectoryProperty
 
-    abstract val cacheRoot: DirectoryProperty
-
+    /**
+     * Whether Dokka is permitted to resolve remote files or links from the network.
+     *
+     * This includes package-lists used for generating external documentation links.
+     * For example, to make classes from the standard library clickable in the generated output.
+     *
+     * Setting this to `true` can significantly speed up build times in certain cases,
+     * but can also worsen documentation quality and user experience.
+     * For example, by not resolving class/member links from your dependencies, including the standard library.
+     *
+     * Note: You can cache files locally and provide them to Dokka as local paths.
+     * See [DokkaSourceSetSpec.externalDocumentationLinks].
+     *
+     * Default: `false`
+     */
     abstract val offlineMode: Property<Boolean>
 
+    /**
+     * Whether to fail documentation generation if Dokka has emitted a warning or an error.
+     * The process waits until all errors and warnings have been emitted first.
+     *
+     * This setting works well with [DokkaSourceSetSpec.reportUndocumented].
+     *
+     * Default: `false`
+     */
     abstract val failOnWarning: Property<Boolean>
 
+    /**
+     * Whether to suppress obvious functions.
+     *
+     * A function is considered to be obvious if it is:
+     *
+     * - Inherited from [kotlin.Any], [kotlin.Enum], [java.lang.Object] or [java.lang.Enum],
+     *   such as `equals()`, `hashCode()`, `toString()` functions.
+     * - Synthetic (generated by the compiler) and does not have any documentation,
+     *   such as `data class` `componentN()` or `copy()` functions.
+     *
+     * Default: `true`
+     */
     abstract val suppressObviousFunctions: Property<Boolean>
 
+    /**
+     * Whether to suppress inherited members that aren't explicitly overridden in a given class.
+     *
+     * Note: This can suppress functions such as `equals()`, `hashCode()`, or `toString()`,
+     * but cannot suppress synthetic functions such as `data class` `componentN()` or `copy()`.
+     * Instead, use [suppressObviousFunctions].
+     *
+     * Default: `false`
+     */
+    abstract val suppressInheritedMembers: Property<Boolean>
+
+    /**
+     * A list of Markdown files that contain module and package documentation.
+     *
+     * The contents of the specified files are parsed and embedded into documentation as module and package descriptions.
+     *
+     * The format of the Markdown files is very specific and must be exactly correct to be readable by Dokka.
+     *
+     * See the
+     * [Dokka Gradle example](https://github.com/Kotlin/dokka/tree/master/examples/gradle-v2/basic-gradle-example)
```

Comment:
Better to use https://kotlinlang.org/docs/dokka-module-and-package-docs.html#pass-files-to-dokka

Reply:
The current kotlinlang Dokka docs are for V1, so I'd prefer to avoid referencing them in V2, until the docs are updated.